js/pages/settings.js

```diff
@@ -144,12 +152,14 @@ export const renderSettingsPage = async () => {
       btnObj.changeNameButton = document.getElementById('changeNameButton');
       btnObj.changeContactInformationButton = document.getElementById('changeContactInformationButton');
       btnObj.changeMailingAddressButton = document.getElementById('changeMailingAddressButton');
-      btnObj.changeEmailButton = document.getElementById('changeEmailButton');
+      btnObj.changeLoginButton = document.getElementById('changeLoginButton');
       showEditButtonsOnUserVerified();
       handleEditNameSection();
       handleEditContactInformationSection();
       handleEditMailingAddressSection();
       handleEditSignInInformationSection();
+      attachTabEventListeners();
+      attachLoginEditFormButtons(optVars.loginEmail, optVars.loginPhone, formVisBools, loginElementArray, btnObj);
```

Comment:
```suggestion
      attachLoginEditFormButtons(optVars.loginEmail, optVars.loginPhone);
```
To be consistent with function definition.

Reply:
My oversight here. `optVars.loginEmail` && `optVars.loginPhone` have scope and can be omitted from the call completely. Updated code:

```javascript
if (userData[cId.verification] == cId.verified) {
      btnObj.changeNameButton = document.getElementById('changeNameButton');
      btnObj.changeContactInformationButton = document.getElementById('changeContactInformationButton');
      btnObj.changeMailingAddressButton = document.getElementById('changeMailingAddressButton');
      btnObj.changeLoginButton = document.getElementById('changeLoginButton');
      showEditButtonsOnUserVerified();
      handleEditNameSection();
      handleEditContactInformationSection();
      handleEditMailingAddressSection();
      handleEditSignInInformationSection();
      attachTabEventListeners();
      attachLoginEditFormButtons();  // <--- args removed
    }
```
```javascript
const attachLoginEditFormButtons = async () => { // <--- args removed
    let removalType = null;

    const modalMap = {
        'Email': optVars.loginEmail ? document.getElementById('confirmationModalEmail') : null, // <---updated arg
        'Phone': optVars.loginPhone ? document.getElementById('confirmationModalPhone') : null // <---updated arg 
    }
// ...unchanged beyond here
```